.ipynb_checkpoints/gensim Quick Start-checkpoint.ipynb

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
` .ipynb_checkpoints/gensim Quick Start-checkpoint.ipynb` - you need to remove this file